lib/generator.js

```diff
@@ -98,7 +98,7 @@
         var self = this;
         
         self._options = options || {};
-        self._config = options.config || {};
+        self._config = self._options.config || {};
         
         _logger.debug("Launching with config:\n%s", JSON.stringify(self._config, null, "  "));
         if (options.photoshopVersion) {
```

Comment:
Nice catch! But, won't the same thing happen right here? How about instead at the beginning of the function we add a line like
```javascript
options = options || {};
```
Then we can just do `self._options = options;` and preserve the following assignment `self._config = options.config || {};`

What do you think?